### PR TITLE
[Do not merge] Fixes js-yaml package vulnerability on gh-1.10.3

### PR DESCRIPTION
**Pull request by msempere**
⚠️  **Do not merge**

Following PR addresses https://github.com/github/data-engineering/issues/1260 by updating a vulnerable package `js-yaml`.
It replicates the same fix **Apache Airflow** has released into their master branch to address the issue: https://github.com/apache/airflow/pull/5384

/cc @github/data-engineering 
/cc https://github.com/github/data-engineering/issues/1260

### CODE REVIEW

File: airflow/www/package.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (alrolorojas) [inline airflow/www/package.json]: It looks like this file is completely new, I was expecting a 1-2 line change. Is there an alternative way they were managing js dependencies on previous versions of Airflow?

Comment (msempere) [reply airflow/www/package.json]: In `1.10.3` there are two UI version, one under `airflow/www`, the one we are currently using, and other under `airflow/www_rbac`, containing this latest one the `package.json` with all the versions.
In newest releases, they have unified all under the same place, `airflow/www` so they seem to be sharing the same `package.json`, but in our case an in our version I don't know if a change on `airflow/www_rbac/package.json` would also affect `airflow/www`

**Comment by suke68** [comment]
@msempere Would you please add some information as to why this is marked as `Do not merge` and what specifically the dependency is?

**Comment by msempere** [comment]
Closing because I believe we can find a better solution to this with fewer changes to our `package.json` file. Will reopen a new one.

**Pull request closed without merging**